### ISSUE TITLE
Bug with `0.0.1-alpha.16` build of `wallet-aggregator-evm`

### ISSUE THREAD

**Issue by tonyjin** [XLabs/wallet-aggregator-sdk#4]
Ran into the following error with `0.0.1-alpha.16` build:
```
ERROR in ../node_modules/@walletconnect/ethereum-provider/dist/index.es.js 242:16-47
Module not found: Error: Can't resolve '@web3modal/standalone' in '/home/tjin/projects/wormhole-connect/node_modules/@walletconnect/ethereum-provider/dist'
```

Not reproducible with `0.0.1-alpha.15`

**Comment by M-Picco** [comment]
Hi! Thanks for reporting this. It seems a dependency down the tree switched that package to a peer dependency (narrowed it down to `@walletconnect/ethereum-provider` [v2.5.0](https://github.com/WalletConnect/walletconnect-monorepo/releases/tag/2.5.0)).

The workaround would be to install `@web3modal/standalone`. I'll look into adding the dependency for the next version.

**Comment by M-Picco** [comment]
Fixed with `0.0.1-alpha.17` release
